### PR TITLE
Add in __call__ method

### PR DESCRIPTION
**Pull request by PythonCoderAS**
## About this pull request

Adds in a __call__ method to `CallbackObject`.

## Changes

See above.

## Checklist

- [x] I've run the `pre_push.py` script to format and lint code.
- [x] I've checked this pull request runs on `Python 3.6.X`.
- [ ] This fixes something in [Issues](https://github.com/eunwoo1104/discord-py-slash-command/issues).
    - Issue:
- [x] This adds something new.
- [ ] There is/are breaking change(s).
- [ ] (If required) Relevant documentation has been updated/added.
- [ ] This is not a code change. (README, docs, etc.)

### CODE REVIEW

**Comment by LordOfPolls** [comment]
This pr doesnt follow our coding style, please run `pre_push.py`

File: discord_slash/model.py [hunk 1]
```diff
@@ -125,6 +125,15 @@ class CallbackObject:
     :ivar __commands_checks__: Check of the command.
     """
 
+    async def __call__(self, *args, **kwargs):
+        """
+        Ref: https://github.com/Rapptz/discord.py/blob/f14e584304d9d6676c90f8aa59d6e4be23d22a10/discord/ext/commands/core.py#L396
+        and https://github.com/discord-py-slash-commands/discord-py-interactions/blob/d68ea8ed00720d50845aac586366145cd4a70f0a/discord_slash/model.py#L207-L209
```
Comment (LordOfPolls) [inline discord_slash/model.py:131]: I think rather than using refs, it would be better to actually write a docstring for this method

**Comment by i0bs** [comment]
Updating this PR. Please use `:class:` or `:meth:` RST doctyping.

**Comment by i0bs** [comment]
This PR is almost a month old now with no response or changes, so I will be closing this due to inactivity.

**Pull request closed without merging**